tests/roots/test-ext-autodoc/target/typed_dict.py

```diff
@@ -0,0 +1,11 @@
+from typing import TypedDict
+
+
+class Parent(TypedDict):
+    #: parent attr1 doc
+    p_attr1: int
+
+
+class Child(Parent, TypedDict):
```

Comment:
Can you add a test case where you don't explicitly inherit from TypedDict? or explain why it's required?

Reply:
Can you also add a test with more attributes being overwritten? for instance:

```py
class Base(TypedDict):
	a: int
	"""attr a base"""

class Inner(Base)
	a: int
	"""attr a inner"""

class Outer(Inner):
	b: int
	"""attr b outer"""
```

I suspect that Outer should have a documented `Outer.a` from `Inner.a` but not with the text of `Base.a`.